### PR TITLE
[sonic-feature.yang] fix check_up_status field type

### PR DESCRIPTION
**Pull request by stepanblyschak**
Signed-off-by: Stepan Blyschak <stepanb@nvidia.com>

#### Why I did it

An issue is encountered when a value "False" is written for a feature in "check_up_status" field, which does not pass YANG validation.

#### How I did it

 We usually use stypes::boolean_type for such fields, even in this YANG model. This custom type, supports "False" value.

#### How to verify it

Write "False" in "check_up_status" field and see if YANG validation passes.

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106
- [ ] 202111

### CODE REVIEW

File: src/sonic-yang-models/yang-models/sonic-feature.yang [hunk 1]
```diff
@@ -90,8 +90,8 @@ module sonic-feature{
                 leaf check_up_status {
                     description "This configuration controls the system ready tool to check
                         the app ready/up status";
-                    type boolean;
-                    default false;
+                    type stypes:boolean_type;
+                    default "false";
```
Comment (qiluo-msft) [inline src/sonic-yang-models/yang-models/sonic-feature.yang:94]: Add unit test?

Comment (stepanblyschak) [reply src/sonic-yang-models/yang-models/sonic-feature.yang:94]: @qiluo-msft Without this fix sonic-utilities submodule update is blocked. There were no UT when this field was added. I suggest to take this fix to unblock submodule update and the owner of new feature adds UT.

Comment (qiluo-msft) [reply src/sonic-yang-models/yang-models/sonic-feature.yang:94]: @wen587 for awareness of blocking issue of sonic-utilities submodule update

**Review by qiluo-msft — APPROVED** [review]

**Comment by liat-grozovik** [comment]
@stepanblyschak please chck if this is needed for 202205 and if so update PR desciption/label

**Pull request merged**